diff --git a/config.def.h b/config.def.h
index c711154..254dede 100644
--- a/config.def.h
+++ b/config.def.h
@@ -5,9 +5,10 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "Iosevka NFM:size=14:antialias=true:autohint=true";
 /* Spare fonts */
 static char *font2[] = {
+	"Apple Color Emoji:size=14:antialias=true:autohint=true",
 /*	"Inconsolata for Powerline:pixelsize=12:antialias=true:autohint=true", */
 /*	"Hack Nerd Font Mono:pixelsize=11:antialias=true:autohint=true", */
 };
